src/suvela/assignments/Assign29.java

```diff
@@ -0,0 +1,37 @@
+/*Assignment - 29 : 26th Nov'2020 
+Find second maximum from array without sorting the array.*/
+package suvela.assignments;
+
+import java.util.Arrays;
+
+public class Assign29 {
+	void findSecondMax(int[] array) {
+		System.out.println("Given array:" + Arrays.toString(array));
+		int max = 0;
+		int secondMax = 0;
+		if (array[0] > array[1]) {
+			max = array[0];
+			secondMax = array[1];
+		} else {
+			max = array[1];
+			secondMax = array[0];
+		}
+		for (int i = 2; i < array.length; i++) {
+			int num = array[i];
+			if (num > max) {
+				secondMax = max;
+				max = num;
+			} else if (num < max)
```

Comment:
you should write
if(num > secondLargest && num != largest) {
	secondLargest = num;
}